src/main/scala/objsets/TweetSet.scala

```diff
@@ -179,15 +230,15 @@ class Cons(val head: Tweet, val tail: TweetList) extends TweetList {
 object GoogleVsApple {
   val google = List("android", "Android", "galaxy", "Galaxy", "nexus", "Nexus")
   val apple = List("ios", "iOS", "iphone", "iPhone", "ipad", "iPad")
-
-  lazy val googleTweets: TweetSet = ???
-  lazy val appleTweets: TweetSet = ???
+  val allTweets = TweetReader.allTweets
+  lazy val googleTweets: TweetSet = allTweets.filter(tweet => google.exists(word => tweet.text.contains(word)))
```

Comment:
```suggestion
  lazy val googleTweets: TweetSet = allTweets.filter(tweet => google.exists(tweet.text.contains))
```
A shorter way to write the same thing :)